test/domain/model/contact_test.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:fluffychat/domain/model/contact/contact.dart';
+import 'package:fluffychat/domain/model/contact/hash_details_response.dart';
+
+void main() {
+  group('calLookupAddress', () {
```

Comment:
I suggest the structure:
- group('calLookupAddress for email')
  - test('no algorithm')
  - test('no pepper')
  - test('algorithm was not supported')
  - test('algorithm not implement')
  - test('happy case')
- group('calLookupAddress for phone')
  - ...

Reply:
for the phone, we will have a lot of corner cases. please try to cover all these cases

Reply:
for phone we need to be standardized
we can use that https://github.com/Shypes/msisdn-sanitizer#misidn-sanitizer
but it require country phone code
any idea to get that ?

Reply:
no worries, I will use this to get default dial code
https://pub.dev/packages/country_codes

Reply:
no i mean the tests, we need to cover the test cases.